internal/provider/be_api_key_resource.go

```diff
@@ -166,9 +175,6 @@ func (r *beApiKeyResource) Read(ctx context.Context, req resource.ReadRequest, r
 	state.Name = types.StringValue(beApiKeyInfo.Name)
 	state.ApiKeyId = types.StringValue(beApiKeyInfo.ApiKeyId)
 	state.ReadOnly = types.BoolValue(beApiKeyInfo.IsReadOnly)
-	// state.ApiKey is omitted, as it is stripped due to containing sensitive data.
```

Comment:
Removed this comment, as now the [BE does not return it at all anyways](https://github.com/PropelAuth/auth/pull/748/commits/cd649bc2f6e4cba93f27eb39f931c43788769ad6)